lib/ansible/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -230,6 +230,13 @@
       - A hash/dictionary of map host parallel devices. C(parallel='{"key":"value", "key":"value"}').
       - Keys allowed are - (parallel[n]) where 0 ≤ n ≤ 2.
       - Values allowed are - C("/dev/parport\d+|/dev/usb/lp\d+").
+  pause:
+    description:
+      - Pause the cluster before getting next C(<vmid>).
+      - Please see issue => https://github.com/ansible/ansible/issues/28545
+      - We recommend using C(loop_control). See https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#loop-control
+    default: 0
+    version_added: "2.7"
```

Comment:
If we add this new option then this is a "Feature PR", and can't be backported to `stable-2.6`.

Reply:
This is an option only to bypass what the  `pause` parameter on `loop_control` already does.
I will delete it from this PR and add it eventually in a future version of this module (Not necessarily ...)